src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputFormattedNoLineWrapping.java

```diff
@@ -0,0 +1,23 @@
+package com.google.checkstyle.test.chapter3filestructure.rule332nolinewrap;
+
+import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.PackageStatementTest; // ok, long imports are allowed
```

Comment:
If I this import statement was never used in the class then formatter would have removed it and we would miss and edge case, so I created an instance of it to test the edge case and maintain similarity between both input files